users/goelashwin36/Task-3/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Quiz</title>
+<link rel="stylesheet" href="css/style.css">
+<script src="js/javascript.js"></script>
+<script
+  src="http://code.jquery.com/jquery-3.3.1.min.js"
+  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+  crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.bundle.min.js"></script>
+</head>
+<body>
+<center><img src="img/qt.png" width="300px" height="200px" ></center>
+
+<div class="container">
+    <div class="row back">
+        <div class="question">
+                1. Which of the following country won Football world Cup maximum times?<br>
+        </div>
+    </div>
+
+        <div class="answer" id="Q1">
+                <div class="form-check">
+                        <input class="form-check-input" type="radio" name="1" id="exampleRadios1" value="0" >
+                        <label class="form-check-label" for="exampleRadios1" name="a">
+                          Germany                        </label>
+                      </div>
+                      <div class="form-check">
+                            <input class="form-check-input" type="radio" name="1" id="exampleRadios1" value="1" >
+                            <label class="form-check-label" for="exampleRadios1" name="b">
+                              Brazil                            </label>
+                          </div>
+                      <div class="form-check">
+                        <input class="form-check-input" type="radio" name="1" id="exampleRadios1" value="0">
+                        <label class="form-check-label" for="exampleRadios1" name="c">
+                          Argentina                        </label>
+                      </div>
+                      <div class="form-check">
+                        <input class="form-check-input" type="radio" name="1" id="exampleRadios1" value="0">
+                        <label class="form-check-label" for="exampleRadios1" name="d">
+                          Italy                        </label>
+                    </div>
+    </div>
+    <div class="row back">
+            <div class="question">
+              2. Who among the following player scores highest number of goals in Footbal World Cup?
```

Comment:
You can always use `padding` to have your UI more cleaner, also if you are working with square borders, give them a little curve on the corners using `border-radius`